server/plugins/TestUGens.cpp

```diff
@@ -44,16 +44,18 @@ struct CheckBadValues : public Unit
 	int		prevclass;
 };
 
+struct Sanitize : public Unit { };
+
 // declare unit generator functions
-extern "C"
```

Comment:
I don't know the difference, but I trust Tim: https://github.com/supercollider/example-plugins/commit/166c18cf3de457731618cb4f8c4bb613e6fab398#commitcomment-17393581